apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -8,47 +8,35 @@ import MultipleChoiceAnswerCell from './MultipleChoiceAnswerCell';
 
 export const COLUMNS = {
   QUESTION: 0,
-  ANSWER_1: 1,
-  ANSWER_2: 2,
-  NOT_ANSWERED: 3,
 };
 
-const answerOptionsOneFormatter = (percentAnsweredOptionOne, {rowData}) => {
-  return (
-      <MultipleChoiceAnswerCell
-        id={rowData.id}
-        percentValue={rowData.percentAnsweredOptionOne}
-        isCorrectAnswer={rowData.optionOneIsCorrect}
-      />
-  );
-};
+const alphabetMapper =  [
+  commonMsg.answerOptionA(),
+  commonMsg.answerOptionB(),
+  commonMsg.answerOptionC(),
+  commonMsg.answerOptionD(),
+  commonMsg.answerOptionE(),
+  commonMsg.answerOptionF(),
+  commonMsg.answerOptionG(),
+];
 
-const answerOptionsTwoFormatter = (percentAnsweredOptionOne, {rowData}) => {
+const answerColumnsFormatter = (percentAnswered, {rowData, columnIndex, rowIndex, property}, index) => {
+  const column = (columnIndex - 1);
+  const cell = rowData.answers[column];
   return (
       <MultipleChoiceAnswerCell
         id={rowData.id}
-        percentValue={rowData.percentAnsweredOptionTwo}
-        isCorrectAnswer={rowData.optionTwoIsCorrect}
-      />
-  );
-};
-
-const notAnsweredFormatter = (percentAnsweredOptionOne, {rowData}) => {
-  return (
-      <MultipleChoiceAnswerCell
-        id={rowData.id}
-        percentValue={rowData.notAnswered}
+        percentValue={cell && `${cell.percentAnswered}%` || '-'}
+        isCorrectAnswer={cell && cell.isCorrectAnswer || false}
```

Comment:
If `cell` is `undefined`, then `cell && cell.isCorrectAnswer` will return `undefined`. I'm pretty sure it would still work without the `false` though, since `undefined` is falsy.

Reply:
Fixed!  Thanks